e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/AwaitHelper.java

```diff
@@ -9,9 +9,9 @@
 
 public class AwaitHelper {
 
-    protected static final int WAIT_FOR_IN_SECONDS = 10;
-    protected static final int POLL_INTERVAL_MS = 100;
-    protected static final int POLL_DELAY_MS = 10;
+    protected static final int WAIT_FOR_IN_SECONDS = 30;
```

Comment:
30 seconds is a long time to wait for a test to time out. Was this change really needed?

Reply:
Sorry, artefact from testing.